src/event-handler.js

```diff
@@ -20,7 +22,55 @@ EventHandler.prototype.logEvent = function(event) {
         }
         window['optimizely'].push(optimizelyEvent);
     }
+
+    if (optimizelyFullStackEvents.events[event.EventName]) {
```

Comment:
rather than checking for `optimizelyFullStackEvents.events....`, I would do:
```suggestion
    // if optimizely full stack is being used
    if (window.optimizelyClientInstance) {
```
as this checks to see if optimizely full stack SDK is being loaded which is what we're trying to do and it also is a bit more clear in the logic flow (perhaps add a comment as well).

Reply:
makes sense!